spec/features/user_reset_password_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature 'User reset password' do
+  scenario 'user forgot password' do
+    user = Fabricate(:user)
+
+    visit login_path
+    click_link 'Forgot password?'
+    expect(page).to have_content('Forgot Password?')
+    fill_in "email", with: user.email_address
+    click_button 'Send Email'
+    expect(ActionMailer::Base.deliveries.last.to).to eq([user.email_address])
+    visit "/password_reset/#{user.token}"
```

Comment:
This is another place you should be using names routes rather than hard coding urls. If you use named routes, you can change the actual url and your test code won't need to change.